code/_helpers/lists.dm

```diff
@@ -718,3 +718,17 @@ var/global/list/json_cache = list()
 	catch(var/exception/e)
 		log_error("Exception during JSON decoding ([json_to_decode]): [e]")
 		return list()
+
+/// Generates a list used to randomize transit animations so they aren't in lockstep.
+/proc/get_cross_shift_list(size)
```

Comment:
this proc is weird.
it seems to just randomly shuffle the shifts list in effect, why not just do

phase_shift_by_x  = list(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14)
...
phase_shift_by_x  = shuffle(phase_shift_by_x) in skybox code?